documentation/latest/dev/guide.md

```diff
@@ -3,24 +3,32 @@ layout: documentation
 title: Developer guide
 ---
 
+## Integration tests
 
-# Integration tests
+### Quick start
 
-## Quick start
+#### Execute integration tests locally
+
+To execute integration tests locally, you need to first start the
+`workflow-service` and `notification-service` services by running the following
+commands:
 
-### Execute integration tests locally
-To execute integration tests locally, you need to first start the `workflow-service` and `notification-service` services by running the following commands:
 ```bash
 ./start_workflow_service.sh &
 ./start_notification_service.sh &
 ```
-Once the services are up and running, you can execute the integration tests by running the following command:
+
+Once the services are up and running, you can execute the integration tests by
+running the following command:
```

Comment:
following up to this point fails for me with:
```
[INFO] 
[ERROR] Failures: 
[ERROR]   PrebuiltWorkFlowTest.runPreBuiltWorkFlow:94
[INFO] 
[ERROR] Tests run: 5, Failures: 1, Errors: 0, Skipped: 0
```
It appears that PrebuiltWorkFlowTest tries to execute a notification task that fails.
Do we need to provide any special hints for the notification service to connect to in this type of run?

Reply:
The error from the workflow-service logs:
```
2023-05-18 10:26:58.677 ERROR 1023017 --- [        task-11] c.r.p.t.n.NotificationWorkFlowTask       : Exception when calling NotificationMessageApi#create:

com.redhat.parodos.notification.sdk.api.ApiException: Message: 
HTTP response code: 404
HTTP response body: {
  "timestamp" : "2023-05-18T07:26:58.675+00:00",
  "status" : 404,
  "error" : "Not Found",
  "path" : "/api/v1/messages"
}

```

Reply:
investigating

Reply:
this fixes the issue:
https://github.com/parodos-dev/parodos/pull/348